.github/workflows/build-packages.yml

```diff
@@ -442,17 +442,16 @@ jobs:
           name: salt-${{ inputs.salt-version }}-onedir-windows-${{ matrix.arch }}.zip
           path: artifacts/
 
+      - name: Code signing with Software Trust Manager
+        if: ${{ steps.check-pkg-sign.outputs.sign-pkgs == 'true' }}
+        uses: digicert/ssm-code-signing@v1.0.0
+
```

Comment:
I think this is reverting a change that we need to sign Windows. The "Setup Certificate" step needs to happen first.